nservicebus/azure-service-bus/configuration/configuration.md

```diff
@@ -0,0 +1,200 @@
+ * `MessagingFactorySettingsFactory(Func<string, MessagingFactorySettings>)`: This factory method allows to override creation of messaging factories.
+ 
+### Message Receivers
+
+The following settings determine how NServiceBus will receive messages from Azure Service Bus entities.
+
+ * `ReceiveMode(ReceiveMode)`: Determines part of the 'transactional behavior'. `ReceiveMode.PeekLock` ensure that messages will be only removed from the queue after processing is done. If processing fails, or takes to long, the message will reappear on the queue for re-processing. Basically emulating transactional rollback. `ReceiveMode.ReceiveAndDelete` will delete the message immediately after receive, meaning there is no rollback when an exception occurs. Defaults to `ReceiveMode.PeekLock`.
+ * `PrefetchCount(int)`: This setting will make the receiver prefetch messages on receive operations, and acts like client side batching. Defaults to 200, meaning that each receive operation will pull in another 200 messages.
+ * `RetryPolicy(RetryPolicy)`: Determines how the receiver should respond on transient connectivity failures. Defaults to `RetryPolicy.Default`, which is an exponential retry under the hood.
+ * `AutoRenewTimeout(TimeSpan)`: When using `ReceiveMode.PeekLock`, the broker will lock a received message for a period specified as `LockDuration(TimeSpan)` on the entity. If processing doesn't end within this time period, a message will reappear on the entity and will be reprocessed. For long running operations this might imply that the operation will be executed multiple times in parallel, which is usually undesirable. This setting allows to extend the lock automatically when processing hasn't finished yet. Ensure that the `TimeSpan` specified here is a little smaller than the `TimeSpan` specified as `LockDuration`.
+
+### Message Senders
+
+The following settings determine how NServiceBus will send messages to Azure Service Bus entities.
+
+ * `RetryPolicy(RetryPolicy)`: Determines how the sender should respond on transient connectivity failures. Defaults to `RetryPolicy.Default`, which is an exponential retry under the hood.
+ * `BackOffTimeOnThrottle(TimeSpan)`: 'As A Service' products will tell there clients to back off under heavy load and so will Azure Service Bus. This setting specifies how long NServiceBus will wait to try sending again in these occurances. Defaults to 10 seconds. Note, by default this backoff period will increase exponentially during retries. 
+ * `RetryAttemptsOnThrottle(int)`: Defines how many times the transport should attempt to retry a send operation when being operated before throwing an exception.
+ * `MaximumMessageSizeInKilobytes(int)`: Specifies how large messages (and batches of messages) can be. This can differ between different types of namespaces. The default value of 256 aligns with Standard namespaces.
```

Comment:
> This can differ between different types of namespaces

I don't see here how I can control it per namespace. Is this setting applied globally to all namespaces?